cmd/kubeadm/app/cmd/init.go

```diff
@@ -32,20 +32,21 @@ import (
 )
 
 var (
-	init_done_msgf = dedent.Dedent(`
+	initDoneMsgf = dedent.Dedent(`
 		Kubernetes master initialised successfully!
 
-		You can connect any number of nodes by running:
+		You can now join any number of machines by running the following on each node:
```

Comment:
"You can now join any number of machines as nodes by running the following on each of them" is clearer IMO. Although the user may not understand what a "node" is...